configs/mconfig.Darwin

```diff
@@ -1,7 +1,7 @@
 # Installation path options.
 
-SBINDIR=/sbin
-MANDIR=/usr/share/man
+SBINDIR=/usr/local/sbin
+MANDIR=/usr/local/share/man
 SYSCONTROLSOCKET=/var/run/dinitctl
```

Comment:
Hi. Thanks for the contribution :)

Hmm, As I read, /var is also protected by SIP. It should be changed to some where else I guess (or is /var/run a exception?)

Reply:
You're correct, `/var` is protected but `/var/run` is an exception.